README.md

```diff
@@ -8,30 +8,48 @@ This repository holds the ReStructuredText (RST) source, and other files, for us
 
-## Using nox
+## Verifying your pull request
 
-This project includes a `nox` configuration to automate checks and other functions.
-You should use `nox` to run checks locally before you submit a pull request.
+We welcome all contributions to Ansible community documentation.
+If you plan to submit a pull request with changes, you should [verify your PR](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#verifying-your-documentation-pr) to ensure it conforms with style guidelines and can build successfully.
 
-Install `nox` using `python3 -m pip install nox` or your distribution's package manager.
+### Running automated tests
 
-Run `nox --list` from the repository root to view available sessions.
+This project includes a `nox` configuration to automate tests, checks, and other functions.
+You can use these automated tests to help you verify changes before you submit a PR.
 
-Run `nox` with no arguments to execute the default sessions.
+1. Install `nox` using `python3 -m pip install nox` or your distribution's package manager.
+2. Run `nox --list` from the repository root to view available sessions.
 
-## Running the spelling check
+Each `nox` session creates a temporary environment that installs all requirements and runs the test or check.
+This means you only need to run one command to perform the test accurately and consistently.
+Here are some `nox` sessions you can run:
 
-This repository uses [`codespell`](https://github.com/codespell-project/codespell) to check for common spelling mistakes in the documentation source.
+* Run `nox` with no arguments to run all available sessions.
+* Run `nox -s clone-core` to clone required parts of the `ansible/ansible` repository as mentioned in the [local build documentation](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally).
+* Run `nox -s "checkers(rstcheck)"` to ensure there are no syntax errors in the reStructuredText source files.
+* Run `nox -s "checkers(docs-build)"` or `nox -s "checkers(docs-build)" -- --relaxed` to verify the docs build.
 
-Run `nox -s spelling` to check spelling.
+> The `nox -s "checkers(docs-build)"` session cleans the generated docs after it runs.
+> If you want to view the generated HTML in your browser, you should build the documentation locally.
 
-Run `nox -s spelling -- -w` to correct spelling errors.
+## Checking spelling
 
-When `codespell` suggests more than one word as a correction, run `nox -s spelling -- -w -i 3` to select an option.
+Use [`codespell`](https://github.com/codespell-project/codespell) to check for common spelling mistakes in the documentation source.
 
-## Updating the dependencies
+* Run `nox -s spelling` to check spelling.
+* Run `nox -s spelling -- -w` to correct spelling errors.
+
+> When `codespell` suggests more than one word as a correction, run `nox -s spelling -- -w -i 3` to select an option.
```

Comment:
I think it would make sense to include this in the bulleted list instead of as a separate blockquote, but that's just me.

Reply:
Yeah that does make sense. Thanks.